packages/front-end/components/Metrics/MetricsList.tsx

```diff
@@ -271,32 +256,21 @@ const MetricsList = (): React.ReactElement => {
     ? metrics.filter((m) => isProjectListValidForProject(m.projects, project))
     : metrics;
 
-  // Searching
-  const filterResults = useCallback(
-    (items: typeof filteredMetrics) => {
-      if (!showArchived) {
-        items = items.filter((m) => {
-          return !m.archived || recentlyArchived.has(m.id);
-        });
-      }
-      items = filterByTags(items, tagsFilter.tags);
-      return items;
-    },
-    [showArchived, recentlyArchived, tagsFilter.tags]
-  );
```

Comment:
There's a bug with the archived filter in the new version.  Once you show archived items, there's no way to unshow them.  Removing the filter still keeps them in the list.

Reply:
Fixed